ee-9/dist/src/test/java/org/wildfly/dist/subsystem/xml/StandardConfigsXMLValidationUnitTestCase.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2010, Red Hat Middleware LLC, and individual contributors
```

Comment:
Copyright date needs an update.

This pr is really great Brian, thanks for pulling this all together in WF for all of us to benefit from!

Reply:
Thanks, @scottmarlow! I'll get the copyright dates corrected.